packages/asc-core/src/Radio/Radio.tsx

```diff
@@ -50,8 +50,10 @@ const Label = styled.label`
   }
 `
 
-const Wrapper = styled.div`
+const Wrapper = styled.span`
   position: relative;
+  display: ${props => props.orientation === 'horizontal' ? 'inline-block' : 'block' };
+  margin-right: ${props => props.orientation === 'horizontal' ? '24px' : '0' };
```

Comment:
For the PoC we can use these arbitrary values like 24px, but eventually we might want to consider using styled-system or sth else. 😄 https://github.com/jxnblk/styled-system